src/blackjack.js

```diff
@@ -5,8 +5,26 @@
  */
 function cardValue(cardName) {
 
+  if (cardName === 'ace') {
+  return 11;
+  }
+
+  if (cardName === 'jack') {
+    return 10;
+  }
+
+  if (cardName === 'queen') {
+    return 10;
+  }
+
+  if (cardName === 'king') {
+    return 10;
+  }
+
+  else {return parseInt(cardName, 10)};
```

Comment:
Parse int will automatically do conversion to base 10 (regular counting numbers) so you don't need to pass in `10` as a second argument.